portfolio-project-matching-app/components/NewLogin.js

```diff
@@ -1,18 +1,17 @@
 import Button from './Button';
 import auth from '../Firebase/auth';
-import { useEffect, useState } from 'react';
 import { useAuth, useAuthUpdate } from '../context/AuthContext';
 
 
-
-const SignInButton = () => {
+const GoogleSignInButton = () => {
     let updateAuth = useAuthUpdate();
     async function signIn(){
-        let result = await auth.signin();
+        let result = await auth.signinWithGoogle();
         updateAuth(result);
+        // Check if user is in user collection using uid
```

Comment:
Is this `todo`?

Reply:
Not anymore. I'll fix it and push up my changes.

Reply:
I was confused about where I should check if user is in DB. I do it in the `signInWithGoogle` function now.

Reply:
Nice!!